test/integ_tests/test_create_quantum_job.py

```diff
@@ -200,7 +201,7 @@ def test_completed_quantum_job(aws_session, capsys):
 
 
 @pytest.mark.xfail(
-    (sys.version_info.major, sys.version_info.minor) != decorator_python_version,
+    (sys.version_info.major, sys.version_info.minor) != (3, 10),
```

Comment:
I'm not sure I understand why the previous setup doesn't work. Could you please add more details on what is causing that to fail?

Reply:
Yes, the fixture doesn't get evaluated inside the decorator, so this comparison is something like `(3, 10) != <pytest.fixture-decorator_python_version>`

Not sure how this didn't get caught in its pr, but there was a lot of back and forth with the method for skipping and looks like the final version missed the mark

Reply:
Does it necessarily need to be a fixture? Can it be instead set to a regular variable?